### PR TITLE
Add an email verification system in order to properly register users #54

### PR DESCRIPTION
**Pull request by ErikGmz**
Added a strict email verification requirement in order to successfully login with user account.

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
+  # config.action_mailer.raise_delivery_errors = true
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = {
+  #   user_name: "apikey",
+  #   password: Rails.application.credentials.sendgrid_api_key,
+  #   domain: "outlook.com",
+  #   address: "smtp.sendgrid.net",
+  #   port: 587,
+  #   authentication: :plain,
+  #   enable_starttls_auto: true
```
Comment (juperameza) [inline config/environments/production.rb]: Don't leave comment d code

**Review by juperameza — CHANGES_REQUESTED** [review]

File: config/environments/production.rb (hunk 1 above)
Comment (ErikGmz) [reply config/environments/production.rb]: Fixed!

File: config/credentials.yml.enc [hunk 2]
```diff
@@ -1 +1 @@
-bSZHF6a1px4IDYW0xPmGsrbwYPruIzTyhUB6xXiEjD1y3BOLYXFeN+GsQJO1P2vUBor8K400gQpVRDr845bQDBhz+9a4+uPC8PHDj+XuWIFyRKBN7891TuSLJtwklZZfk5E9UoyW8LbQsG3GKC9C4S2NrFvNCTwrSSk2/CfTXcC6cI92RlBs0TiYPo4z29vS3pcj8Yqrt1QUPh2YJDPZUwZycy691gCQspMC1Bp488FK/Gdzs8Avz/pm7JCjbcUyRPXUmLbKq8dOjqPYdzpf9BGBamugX02/y5E+XwyUXjT5Lcjqn4zwbqv8RFVpB6Y9kkjUqrv+0QkjfFy/dOzzY/sa/itDnURJNizaKMmfPGe1Pnt9A5yssiDmOOZhmNdnXWS6h5ZtDBmFGluRKoW1VBqeE6Wc--uCQnF/VvUwGE+1sP--dmXXZckAQEBwVYvMCbTw3Q==
\ No newline at end of file
+9a34P6rlO2y1tBm134h72IlJ5g/3o1iN1bf7j4sidhZYaKy/sQq+me77YhqgjBGPJUBHtVMhEBKEsZKOLgHGn6XFCOMgkrDqOg/CFEBRTvI6acE0C/KPS71xqZAWb65uYypFe/qChGhiIXyw2QHG/88lGxU9Mii5dK3PcHR4jn4S9n1qlbPCJeS6QVotUNtvy5F0a5YEuXeSGSWW3n+vbWuriCMsRYdN4zDtHghw5ITL7IKiKNGn1GBYwF2+zUxTsLA34z+9Gu4lNujUN84GUNEK1px8aUGMnk5xjr5RbUD+VjFmfQ6Pk+ONz1wZN5F348JwuFh2FVKejz+dv1sw5p4UPDu7PNJ85TU5AYPbK2J6bGYq7sgqSYawWhKttZ4eUlxL9Sv+dt5s/RkzCVFXuIrTQYberSClquTaRTU08YEyNxvLn9qfBw1Ra2TpEgVMkjU/BEsOaWE0tLPdEexYQU8byVpgJoGmiVm3Gfa9zyHv+SwcmXQYFbkvTyU+Tp+qTsTIEYT6zNH4ISrm5pi6Ks7s1NMU33JehU2beUVnSHY7--ClTYQiOi1S5NimWi--o7znBCnbHDbuDhHuUZge6g==
```
Comment (MauricioNC) [inline config/credentials.yml.enc]: Add an empty line at the end of your file

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: config/credentials.yml.enc (hunk 2 above)
Comment (ErikGmz) [reply config/credentials.yml.enc]: Fixed!

**Review by MauricioNC — APPROVED** [review]

File: config/credentials.yml.enc (hunk 2 above)
Comment (ErikGmz) [reply config/credentials.yml.enc]: The file contains encrypted API keys. It appears that adding an empty line prevents the application from properly decrypting it, resulting in an ActiveSupport::MessageEncryptor::InvalidMessage exception. Therefore, we had to remove that empty line.

**Review by juperameza — APPROVED** [review]

**Pull request merged**